src/notation/view/notationswitchlistmodel.cpp

```diff
@@ -37,6 +37,18 @@ void NotationSwitchListModel::load()
 {
     TRACEFUNC;
 
+    dispatcher()->reg(this, "next-tab",    [this]() { navigateToNextTab(); });
+    dispatcher()->reg(this, "prev-tab",    [this]() { navigateToPrevTab(); });
+    dispatcher()->reg(this, "first-tab",   [this]() { navigateToSpecificTab("first-tab"); });
+    dispatcher()->reg(this, "second-tab",  [this]() { navigateToSpecificTab("second-tab"); });
+    dispatcher()->reg(this, "third-tab",   [this]() { navigateToSpecificTab("third-tab"); });
+    dispatcher()->reg(this, "fourth-tab",  [this]() { navigateToSpecificTab("fourth-tab"); });
+    dispatcher()->reg(this, "fifth-tab",   [this]() { navigateToSpecificTab("fifth-tab"); });
+    dispatcher()->reg(this, "sixth-tab",   [this]() { navigateToSpecificTab("sixth-tab"); });
+    dispatcher()->reg(this, "seventh-tab", [this]() { navigateToSpecificTab("seventh-tab"); });
+    dispatcher()->reg(this, "eighth-tab",  [this]() { navigateToSpecificTab("eighth-tab"); });
+    dispatcher()->reg(this, "last-tab",    [this]() { navigateToSpecificTab("last-tab"); });
```

Comment:
I would alter the `navigateToSpecificTab` method to take an `int index` rather than an action code. 
Inside that method, you would need to do some sanity checks. (`if (index < 0 || index >= m_notations.size()) { return; }`
And here, you would need to write the index instead of the action code. Shouldn't be complicated though, for example these last two lines might become
```suggestion
    dispatcher()->reg(this, "eighth-tab",  [this]() { navigateToSpecificTab(7); });
    dispatcher()->reg(this, "last-tab",    [this]() { navigateToSpecificTab(m_notations.size() - 1); });
```

Reply:
Updated.